app/views/posts/_post.html.erb

```diff
@@ -0,0 +1,3 @@
+<a href= <%= url %>/<%= post.id %> id="post_link"><h4>Post #<%= index %></h4>
+<p><%= post.text.first(150) %></p>
+<p id="comments-likes">Comments: <%= post.comments_counter %>, Likes: <%= post.likes_counter %></p></a>
```

Comment:
- [ ] @SekoViper kindly ensure to use the most_recent_comments method to render 5 most recent comments in your post page and add a link to the post-show page for each post, Also ensure your design matches the wireframe design given, below is a comparison for better understanding:


Wireframe Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/225418948-b800346f-2c36-48e2-9d06-9f61eda3f69c.png) |  ![image](https://user-images.githubusercontent.com/61408860/225419090-2b281ab7-1819-499e-8ae4-e31d2c706e1c.png)